src/model/FaceliftEnum.cpp

```diff
@@ -32,10 +32,14 @@
 #include <QDebug>
 
 namespace facelift {
+    
+namespace Enum {
 
-void onAssignFromStringError(const QString &s)
+void fromStringError(const QString &string)
```

Comment:
bad function name. Either the function name indicates what the function does (with a verb), or it indicates when it is called (onXXXXX).

Reply:
agree!
how about raiseFatalError ?